<launch>

  <!-- Giao tiếp với traffic -->
  <!-- <node name="stiClient" pkg="agvtraffic_simulation" type="stiClient.py"/> -->

  <!-- Nhận lộ trình và lỗi từ stiControl_fake -->
  <node name="stiClient4_fake" pkg="agvtraffic_simulation" type="stiClient4_fake.py"/>

  <node name="stiDevice4_fake" pkg="agvtraffic_simulation" type="stiDevice_fake.py" output="screen" >
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot4.yaml" />
  </node>

  <!-- Điều phối chung -->
  <node name="stiControl4_fake" pkg="agvtraffic_simulation" type="stiControl_fake.py">
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot4.yaml" />
  </node>

  <!-- Điều hướng AGV  -->
  <node name="goal4_control" pkg="agvtraffic_simulation" type="goal_control.py" >
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot4.yaml" />
  </node>

  <!-- AGV - TF -->
  <node name="agv4_sim" pkg="agvtraffic_simulation" type="agv_sim.py" respawn="false" >
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot4.yaml" />
  </node>

</launch>